### DISCUSSION TITLE
How do I compare the datetimefield in Serenity Core ?

### DISCUSSION THREAD

**Discussion by SebnemSeymen** [serenity-is/Serenity#6749] [Help]
I'm trying to write the following code in the repository:

if ((Authorization.UserDefinition as UserDefinition).IsInRole(1002))
{
query.Where(
new Criteria(MyRow.Fields.CreatedOn) < new DateTime(MyRow.Fields.CreatedOn).AddDate(7))
);
}

authorized person can see only one week after the registration date. After a week, the record will not be listed. We can do this with addDay(7) in #c. But I don't know what is the correct syntax for serenity. I will be glad if you help.

**Comment by VictorTomaili** [comment]
You do not need to create new criteria. Just try to use field directly. 
The CreatedOn date should be later than 7 days before. 

`.Where(MyRow.Fields.CreatedOn > DateTime.Now.AddDays(-7))`

**Reply by SebnemSeymen** [reply]
thank you Mr. Victor,thats work for me. Can I ask one more question to you?
